api/v1alpha1/tls_types.go

```diff
@@ -131,3 +135,55 @@ type ClientValidationContext struct {
 	// +optional
 	CACertificateRefs []gwapiv1.SecretObjectReference `json:"caCertificateRefs,omitempty"`
 }
+
+// Configure Envoy proxy how to handle the x-forwarded-client-cert (XFCC) HTTP header.
+type ForwardClientCert struct {
+	// Envoy Proxy mode how to handle the x-forwarded-client-cert (XFCC) HTTP header.
+	// +optional
+	Mode *ForwardMode `json:"mode,omitempty"`
+
+	// Specifies the fields in the client certificate to be forwarded on the x-forwarded-client-cert (XFCC) HTTP header
+	// +kubebuilder:validation:MaxItems=5
+	// +optional
+	Set []ClientCertData `json:"set,omitempty"`
```

Comment:
@envoyproxy/gateway-maintainers is there a better keyword to use here instead of `Set` ?

Reply:
how does `CertDetailsToAdd` sound ?

Reply:
works, changing set to CertDetailsToAdd